greenlight/tests/functional/keyboard_shortcuts/test_browser_window.py

```diff
@@ -39,9 +39,11 @@ def test_search_field(self):
 
         keys = [self.keys.ACCEL]
         if self.marionette.session_capabilities['platformName'] == 'LINUX':
-            keys.append(self.browser.get_localized_entity('searchFocusUnix.commandkey'))
+            keys.append(self.browser.get_localized_entity(
+                        'searchFocusUnix.commandkey'))
```

Comment:
Most of the necessary changes on this PR are temporarily and will go away soon. We haven't had any problems with 80 char limits in the past years, so not sure what your experience was. Lets ask @davehunt how it was working for gaia ui tests, and which coding styles especially line length they prefer.